emsdk.py

```diff
@@ -2564,8 +2566,8 @@ def get_env_vars_to_add(tools_to_activate):
   return env_vars_to_add
 
 
-def construct_env(tools_to_activate):
-  return construct_env_with_vars(get_env_vars_to_add(tools_to_activate))
+def construct_env(tools_to_activate, system, user):
```

Comment:
How about using `persistent` and `system` as the names for these args everywhere to match the names that we use on the command line?  (since they map 1-to-1).

Reply:
I tried to use `persistent` only on the top-level function. From the Windows point of view, persistent means `user`, so using that is a clearer abstraction. I did this so we can safely move these functions to other files/libraries later. I kept `arg_persistent` however on the top-level function so it matches the name of the flag.